packages/vue/src/components/molecules/SfColorPicker/SfColorPicker.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div class="sf-color-picker" :class="{ 'sf-color-picker--active': isOpen }">
+    <transition name="sf-color-picker" mode="out-in">
+      <div v-if="!isOpen" key="1" class="sf-color-picker__open">
```

Comment:
@martaradziszewska IMO we should change number on key to some string value, what do you think

Reply:
`sf-color-picker__open` when I reed CSS I've been a little bit confused. `__open` sounds like state not like element @martaradziszewska